### PR TITLE
add grafana panel for applying unpersisted log

### LINKED ISSUE #16820: missing grafana panels for "apply unpersisted raft logs"

### PR DESCRIPTION
**Pull request by glorv**
### What is changed and how it works?

Issue Number: Close #16820, ref #16717

What's Changed:

```commit-message
- Change `RAFT_DISABLE_UNPERSISTED_APPLY_GAUGE` to `RAFT_ENABLE_UNPERSISTED_APPLY_GAUGE` because it's hard to calculate the previous correctly.
- Add two new grafana panels for "Apply unpersisted raft logs"
```

New panels snapshot:

image

### Related changes

- [ ] PR to update `pingcap/docs`/`pingcap/docs-cn`:
- [ ] Need to cherry-pick to the release branch

### Check List

Tests 

- [x] Unit test
- [ ] Integration test
- [ ] Manual test (add detailed scripts or steps below)
- [ ] No code

Side effects

- [ ] Performance regression: Consumes more CPU
- [ ] Performance regression: Consumes more Memory
- [ ] Breaking backward compatibility

### Release note

```release-note
None
```

### CODE REVIEW

**Comment by glorv** [comment]
@Connor1996 @gengliqi PTAL, thanks

File: components/raftstore/src/store/peer.rs [hunk 1]
```diff
@@ -1211,7 +1211,7 @@ where
             > 0
         {
             self.raft_group.raft.set_max_apply_unpersisted_log_limit(0);
-            RAFT_DISABLE_UNPERSISTED_APPLY_GAUGE.inc();
+            RAFT_ENABLE_UNPERSISTED_APPLY_GAUGE.dec();
```
Comment (Connor1996) [inline components/raftstore/src/store/peer.rs:1214]: If one peer is removed, `dec()` wouldn't be called.

Comment (glorv) [reply components/raftstore/src/store/peer.rs:1216]: fixed, add try dec in `on_destroy_peer`.

**Comment by glorv** [comment]
@Connor1996 @gengliqi  PTAL, thanks

**Review by tonyxuqqi — APPROVED** [review]

**Pull request merged**